php/documentation.md

```diff
@@ -45,7 +49,7 @@ $dsn = 'crate:SERVER_IP:4200';
 $conn = new Crate\PDO\PDO($dsn, null, null, null);
 ```
 
-As Crate doesn't support authentication, the other parameters can be left null.
+As CrateDB doesn't support authentication, the other parameters can be left null.
```

Comment:
We actually do support auth now https://crate.io/docs/pdo/en/latest/connect.html#get-a-connection

Reply:
Thank you. b7454f607 improves the documentation once more, and also addresses this topic.